docker/app_img/Dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG UBUNTU_BASE="ubuntu:22.04"
+
+FROM ${UBUNTU_BASE} AS image_build
+
+ARG OTACLIENT_WHL
+COPY ./${OTACLIENT_WHL} /tmp
```

Comment:
Can I change this part to create whl by hatch in Dockerfile in the later PR?

Reply:
Thank you, understood. The original motivation is to use same environment to avoid issues, especially related to CPU architecture difference between whl and docker for arm.
I will try to separate the steps and use same environment.